src/neo/SmartContract/Iterators/KeyValueIterator.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2015-2021 The Neo Project.
```

Comment:
you might want to use 2022 if this is a new file

Reply:
Done